index.bs

```diff
@@ -3501,9 +3501,19 @@ When verifying a given {{PublicKeyCredential}} structure (|credential|) and an {
     initiated, verify that <code>|credential|.{{Credential/id}}</code> identifies one of the [=public key credentials=] that were
     listed in {{PublicKeyCredentialRequestOptions/allowCredentials}}.
 
-1. If <code>|credential|.{{PublicKeyCredential/response}}.{{AuthenticatorAssertionResponse/userHandle}}</code> is present, verify
-    that the user identified by this value is the owner of the [=public key credential=] identified by
-    <code>|credential|.{{Credential/id}}</code>.
+1. Identify the user being authenticated and verify that this user is the owner of the [=public key credential source=]
```

Comment:
The RP _does not_ itself have access to the [=public key credential source=], yes?   Should that actually be [=public key credential=]?

Reply:
Indees the RP does not, but I figured that `PublicKeyCredential.id` identifies a _credential source_ and not a _credential_. See top-level comment for more...